frontend/packages/integration-tests-cypress/cy-command-additions/nav.ts

```diff
@@ -0,0 +1,22 @@
+export {}; // needed in files which don't have an import to trigger ES6 module usage
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
+  namespace Cypress {
+    interface Chainable<Subject> {
+      clickNavLink(path: [string, string]): Chainable<Element>;
+    }
+  }
+}
+
+// any command added below, must be added to global Cypress interface above
+
+Cypress.Commands.add('clickNavLink', (path) => {
+  cy.get('.pf-c-nav > .pf-c-nav__list > .pf-c-nav__item').within(() => {
```

Comment:
It's better if we can use `data-test` or `data-cy` attributes instead of CSS classes

Reply:
fixed, started using `data-test`

Reply:
> fixed, started using `data-test`

Un-resolving as this instance is not fixed.

Reply:
> Dave> Yes, it is still called out in a comment bullet list I have, but yes, still on the list. Added to PR description todo list.

That's fine, but it makes it harder to review if you resolve the comment before it's addressed.

Reply:
sorry about that, now it is fixed.